q2_sample_classifier/plugin_setup.py

```diff
@@ -55,6 +57,51 @@ def sniff(self):
     BooleanSeriesFormat)
 
 
+Predictions = SemanticType(
+    'Predictions', variant_of=SampleData.field['type'])
+
+
+class PredictionsFormat(model.TextFileFormat):
+    def sniff(self):
```

Comment:
`sniff` is deprecated in favor of `_validate`. Check out the [`FastqGzFormat` for an example](https://github.com/qiime2/q2-types/blob/master/q2_types/per_sample_sequences/_format.py#L153-L159) (the `_validate_` hook accepts a `level` argument that can have the value `min` or `max`).

Reply:
Also, should this format validate some kind of header values?